packages/client/hmi-client/src/page/Home.vue

```diff
@@ -47,10 +47,18 @@
 
 			<!-- Tab section: My projects, Public projects, Sample projects -->
 			<section class="menu">
-				<TabView>
+				<TabView @tab-change="tabChange($event)">
 					<TabPanel v-for="(tab, i) in projectsTabs" :header="tab.title" :key="i">
 						<section class="filter-and-sort">
-							<div v-if="!isEmpty(tab.projects)">
+							<div>
+								<InputText
+									v-model="searchProjects"
+									placeholder="Search for projects"
+									id="searchProject"
+									class="p-inputtext"
+								/>
+							</div>
+							<div>
 								<!-- TODO: Add project search back in once we are ready
```

Comment:
This needs to be removed if you are adding the search back.
Isn't this be done in another PR?

Reply:
Looks like I found what happened to my old commits I was talking to you about in the morning. 
trying to clean this branch up now